reference-service/src/main/resources/db/migration/common/V3.70_create_funding_sub_type_table.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE FundingSubType (
+  `uuid` varchar(36) NOT NULL,
+  `code` varchar(255) NOT NULL,
+  `label` varchar(255) NOT NULL,
+  `parentId` bigint(20) NOT NULL,
+  PRIMARY KEY (`uuid`),
+  UNIQUE KEY idx_uuid_unique (`uuid`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TRIGGER before_insert_FundingSubType
+  BEFORE INSERT ON FundingSubType
+  FOR EACH ROW
+  SET new.uuid = uuid();
```

Comment:
I'm wondering if `uuid()` can be called in the below script: `insert into FundingSubType values (uuid(), ......)`

Reply:
Do you mean instead of adding this trigger? appears to work both ways so up to you

Reply:
The trigger appeared to be the thing breaking the flyway migration, so removed